seonoh/ArchitectureStudy/app/src/main/java/com/example/seonoh/seonohapp/CoinAdapter.kt

```diff
@@ -0,0 +1,29 @@
+package com.example.seonoh.seonohapp
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.seonoh.seonohapp.model.UseCoinModel
+
+class CoinAdapter : RecyclerView.Adapter<CoinItemViewHolder>() {
+
+    private val mData: ArrayList<UseCoinModel> by lazy { ArrayList<UseCoinModel>() }
```

Comment:
초기화 지연을 사용하는 이유에 대해 좀 더 생각을 해봤는데
메모리를 효율적으로 사용하기 위한 부분이 큰 것 같아요
여기서는 바로 객체를 생성하는게 더 좋을 것 같습니다
```suggestion
    private val mData = mutableListOf<UseCoinModel>()
```

Reply:
네 감사합니다 !

Reply:
@seonoh 
반영 커밋 남겨주세요

Reply:
네 감사합니다 !
7896ebf